encoding/resampling.md

```diff
@@ -434,7 +492,7 @@ with shifted kernels functions.
 
 ### Chroma shifting 
 
-When going from 4:2:0 subsampling
+When going from 4:2:0 [subsampling][subsampling]
```

Comment:
when the link name is the same as the text, you can use empty brackets, i.e. `[subsampling][]`.